### PR TITLE
Change DaprID to AppID for consistency

### PR DESCRIPTION
**Pull request by pruthvidhodda**
# Description

Changed all occurrences of DaprID to AppID (case insensitive) for consistency. 

## Issue reference

Please reference the issue this PR will close: [#_[1036]_](https://github.com/dapr/dapr/issues/1036)

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [x] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation
* [ ] Specification has been updated
* [ ] Provided sample for the feature

### CODE REVIEW

File: pkg/runtime/cli.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ func FromFlags() (*DaprRuntime, error) {
 	appProtocol := flag.String("protocol", string(HTTPProtocol), "Protocol for the application: gRPC or http")
 	componentsPath := flag.String("components-path", DefaultComponentsPath, "Path for components directory. Standalone mode only")
 	config := flag.String("config", "", "Path to config file, or name of a configuration object")
-	daprID := flag.String("dapr-id", "", "A unique ID for Dapr. Used for Service Discovery and state")
+	appID := flag.String("app-id", "", "A unique ID for Dapr. Used for Service Discovery and state")
```
Comment (pruthvidhodda) [inline pkg/runtime/cli.go]: Should we change the usage description as well?

Comment (yaron2) [reply pkg/runtime/cli.go]: No, this is still descriptive of what it means for Dapr.

**Comment by youngbupark** [comment]
@pruthvidhodda please resolve the conflicts

**Comment by pruthvidhodda** [comment]
Done, please review it

File: pkg/runtime/cli.go (hunk 1 above)
Comment (pruthvidhodda) [reply pkg/runtime/cli.go]: Isn't it an ID for the application? [Reference](https://github.com/dapr/cli/blob/master/docs/reference/dapr-run.md)

Comment (yaron2) [reply pkg/runtime/cli.go]: Well, it's an ID for both. They are one in the same.

**Pull request merged**